bin/fenix-qr-codes.js

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+/* eslint-disable global-require, no-console */
+
+const path = require('path');
+
+const fetch = require('node-fetch');
+const QRCode = require('qrcode');
+
+const rootDir = path.join(__dirname, '..');
+const distDir = path.join(rootDir, 'dist');
+const lang = 'en-US';
+
+(async () => {
+  const res = await fetch(
+    `https://addons.mozilla.org/api/v5/addons/search/?app=android&promoted=recommended&type=extension&lang=${lang}`
+  );
+  const { results: addons } = await res.json();
+
+  await Promise.all(
+    addons.map(({ id, url }) => {
+      const filePath =   path.join(distDir, `${id}.png`);
+      return QRCode.toFile(filePath, url.replace(`/${lang}`, ''));
```

Comment:
Aren't we going to want some analytics parameters to measure usage ?

Reply:
Yes, we will. This whole project is kind of on hold for now, but I figured I might as well move forward with creating this utility. Once we know exactly how it needs to work it can be updated. How about I create a follow up to add params to the URL once we know exactly what we need?